src/pages/AllCoins/index.js

```diff
@@ -1,11 +1,91 @@
 import React from 'react'
+import axios from 'axios'
+import {
+  ChartPrice,
+  ChartVolume,
+  ChartsContainer,
+  ChartContainerPrice,
+  ChartContainerVolume,
+  CoinContainer,
+  Container,
+  ContentContainer,
+  H1,
+} from './AllCoins.css'
+import BitcoinLineChart from 'components/BitcoinLineChart'
+import BitcoinBarChart from 'components/BitcoinBarChart'
+import { ChartLegendPrice } from 'components/ChartLegendPrice'
+import { ChartLegendVolume } from 'components/ChartLegendVolume'
+import { CoinsTable } from 'components/CoinsTable'

```

Comment:
fix the imports

Reply:
Refactored all folders / components / imports to follow this.